tests/transactional/trup_smoke.pm

```diff
@@ -21,6 +21,11 @@ sub action {
     $reboot //= 1;
     record_info('TEST', $text);
     trup_call($target);
+
+    if ($target =~ /bootloader/ && get_var('FLAVOR') =~ m/-encrypted/i) {
+        record_info("workaround for bsc#1228126");
```

Comment:
If it's a workaround, shouldn't it still be using `record_soft_failure` ?

Reply:
Good point. I am not sure if it should be `record_soft_failure`  because method is given in https://bugzilla.suse.com/show_bug.cgi?id=1228126#c5.  Compare to https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/perl_bootloader.pm#L53 , I didn't use `record_soft_failure` here.

Reply:
> I think we can ask for developer's feedback in bug comments bsc#12288126 for your test result before merging this PR. and they can decide if it is required or a workaround

Added. Thanks

Reply:
Developer said 
```
The sealed key is supposed to be updated by %post script of shim or grub2. However, SL-Micro installs the bootloader in a different way, so the shim-install script may reinstall bootloader in the wrong place and it results in the missing key. 
```
So I used record_soft_failure